Applications/FileIO/GocadIO/GocadAsciiReader.cpp

```diff
@@ -397,7 +504,62 @@ bool GocadTSurfaceReader::parseNodes(
     return false;
 }
 
-bool GocadTSurfaceReader::parseElements(
+bool GocadAsciiReader::parseLineSegments(
+    std::ifstream& in,
+    std::vector<MeshLib::Node*>& nodes,
+    std::vector<MeshLib::Element*>& elems,
+    std::map<std::size_t, std::size_t> const& node_id_map,
+    MeshLib::Properties& mesh_prop)
+{
+    std::string keyword;
```

Comment:
Please constrain variables to minimum possible scope.

Reply:
:heavy_check_mark:

Reply:
Same goes for `data` array.

Reply:
:heavy_check_mark: